src/diffusers/pipelines/audioldm/convert_from_ckpt.py

```diff
@@ -0,0 +1,942 @@
+# coding=utf-8
```

Comment:
Sorry actually can we instead move this out of `src/diffusers`? For SD we exceptionally moved it in because the community asked for it, but it's a bit of a burden to maintain such scripts :sweat_smile:

Reply:
Resolved in https://github.com/huggingface/diffusers/pull/2232/commits/6bc6a75fb54d4b0ecb35e021f0dfedc248ba0132. Ok to keep it in `scripts` like most of the other pipelines?